config/default.yml

```
@@ -181,8 +181,9 @@ Rails/BulkChangeTable:
     - db/migrate/*.rb
 
 Rails/ContentTag:
-  Description: 'Use `tag` instead of `content_tag`.'
+  Description: 'Use `tag.something(` instead of `tag(:something`.'
```

IMHO, it's strange to write only the opening parenthesis.

```suggestion
  Description: 'Use `tag.something` instead of `tag(:something)`.'
```